pkg/plugin/ws_data_proxy.go

```diff
@@ -60,24 +60,10 @@ func (wsdp *wsDataProxy) readMessage() {
 			_, message, err := wsdp.wsConn.ReadMessage()
 
 			if err != nil {
-				if websocket.IsCloseError(err, websocket.CloseGoingAway) {
-					log.DefaultLogger.Error("Disconnection Error", "error", err.Error())
-					sendErrorFrame(fmt.Sprintf("%s: %s", "Disconnection Error", err.Error()), wsdp.sender)
-
-					wsdp.wsConn, err = wsdp.wsConnect()
-					if err != nil {
-						log.DefaultLogger.Error("Reconnection Error", "error", err.Error())
-						sendErrorFrame(fmt.Sprintf("%s: %s", "Reconnection Error", err.Error()), wsdp.sender)
-					}
-
-				} else {
-					// it can be either antoher possible RFC close errors or more generic errors
-					log.DefaultLogger.Error("Failed Read Message Error", "error", err.Error())
-					sendErrorFrame(fmt.Sprintf("%s: %s", "Read WebSocket Error", err.Error()), wsdp.sender)
-
-					break
-				}
+				msg := "Check the host, path, headers and query parameters and make sure they're correct."
+				log.DefaultLogger.Error("Error reading the websocket", "error", err.Error()+msg)
 				time.Sleep(time.Second * 3)
+				sendErrorFrame(fmt.Sprintf("%s: %s. %s", "Error reading the websocket", err.Error(), msg), wsdp.sender)
```

Comment:
After an error here, shouldn't we `break` the loop? `break` was being called on non-close errors before this change.